test/PushNotificationTests/APITests.cpp

```diff
@@ -36,36 +38,56 @@ namespace Test::PushNotifications
             TEST_CLASS_PROPERTY(L"RunAs:Class", L"RestrictedUser")
         END_TEST_CLASS()
 
+        static std::wstring GetDeploymentDir()
+        {
+            WEX::Common::String testDeploymentDir;
+            WEX::TestExecution::RuntimeParameters::TryGetValue(L"TestDeploymentDir", testDeploymentDir);
+            return reinterpret_cast<PCWSTR>(testDeploymentDir.GetBuffer());
```

Comment:
What is the returned type of GetBuffer()? Byte array?

Reply:
Yes it returns a byte array.